repositories.bzl

```diff
@@ -1,6 +1,15 @@
 _CIVETWEB_BUILD_FILE = """
 licenses(["notice"])  # MIT license
 
+config_setting(
```

Comment:
@drigz Is that the proper place for config settings? Would root `BUILD` file be the better place? If yes, how could I reference them?

Reply:
IMO it's great as it is. Since they're only used in one place, it's clearer to declare them where they're used.

For future reference, if they were in the root BUILD file you could refer to them with @com_github_jupp0r_prometheus_cpp//:darwin.